app/build.gradle

```diff
@@ -116,4 +122,5 @@ dependencies {
     testImplementation 'org.mockito:mockito-core:1.10.19'
     testImplementation 'org.hamcrest:hamcrest-library:1.3'
     testImplementation "com.squareup.leakcanary:leakcanary-android-no-op:$leak_canary_version"
+    testCompile "org.mockito:mockito-core:2.+"
```

Comment:
I think you already changed this in another branch, but you can remove if you want.